<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainingDayView</class>
 <widget class="QWidget" name="TrainingDayView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Intewały</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QSpinBox" name="m_pCalories">
        <property name="maximum">
         <number>4000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Kadencja:&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[spm]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateTimeEdit" name="m_pDuration">
        <property name="displayFormat">
         <string>HH:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Czas w IV strefie:&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[hh:mm]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Czas w III strefie:&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[hh:mm]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QSpinBox" name="m_pAvgPulse">
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="2" column="9">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Czas w V strefie:&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[hh:mm]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="10">
       <widget class="QSpinBox" name="m_pCadence">
        <property name="maximum">
         <number>300</number>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Maksymalne tętno:&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[bpm]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Kalorie:&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[kcal]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Średnie tętno:&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[bpm]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Czas trwania:&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[hh:mm:ss]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateTimeEdit" name="m_pFirstZoneTIme">
        <property name="displayFormat">
         <string>HH:mm</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QDateTimeEdit" name="m_pSecondZoneTIme">
        <property name="displayFormat">
         <string>HH:mm</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QSpinBox" name="m_pMinPulse">
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Średnie tempo:&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[mm:ss]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Czas w I strefie:&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[hh:mm]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="8">
       <widget class="QDateTimeEdit" name="m_pFourthZoneTIme">
        <property name="displayFormat">
         <string>HH:mm</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QSpinBox" name="m_pDistance">
        <property name="maximum">
         <number>30000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Dystans:&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[m]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QTimeEdit" name="m_pAvgPace">
        <property name="displayFormat">
         <string>mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="3" column="10">
       <widget class="QPushButton" name="m_pAddIntervalButton">
        <property name="text">
         <string>Dodaj</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Czas w II strefie:&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[hh:mm]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="10">
       <widget class="QDateTimeEdit" name="m_pFifthZoneTIme">
        <property name="displayFormat">
         <string>HH:mm</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QSpinBox" name="m_pMaxPulse">
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Minimalne tętno:&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[bpm]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QDoubleSpinBox" name="m_pStepLength">
        <property name="maximum">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Długość kroku:&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[m]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QDateTimeEdit" name="m_pThirdZoneTIme">
        <property name="displayFormat">
         <string>HH:mm</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="m_pSaveTrainingButton">
     <property name="text">
      <string>Zapisz i zamknij</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Dzień treningowy</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tętno spoczynkowe:&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[bpm]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="m_pHrRest">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Waga:&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[kg]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="m_pWeight">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Jakość snu:&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[wybór]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QComboBox" name="m_pSleepLevel">
        <item>
         <property name="text">
          <string>Bardzo dobra</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dobra</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Normalna</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Zła</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bardzo zła</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tętno progowe:&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[bpm]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QSpinBox" name="m_pTresholdPulse">
        <property name="maximum">
         <number>300</number>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tempo progowe:&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[mm:ss]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QDateTimeEdit" name="m_pTresholdPace">
        <property name="displayFormat">
         <string>mm:ss</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Trening</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="10">
       <widget class="QDateTimeEdit" name="m_pAvgPaceTraining">
        <property name="displayFormat">
         <string>mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Średnie tempo:&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[mm:ss]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Średnie tętno:&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[bpm]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Nazwa:&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[tekst]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QSpinBox" name="m_pAvgPulseTraining">
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QComboBox" name="m_pMood">
        <item>
         <property name="text">
          <string>Bardzo dobre</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dobre</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Normalne</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Złe</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bardzo złe</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="m_pTrainingName"/>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Godzina treningu:&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[hh:mm]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Samopoczucie:&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[wybór]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QDateTimeEdit" name="m_pTrainingTime">
        <property name="displayFormat">
         <string>HH:mm</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="4">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Plan</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QPlainTextEdit" name="m_pPlanTextEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="5" colspan="8">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Realizacja</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QPlainTextEdit" name="m_pRealisationTextEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_26">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Efektywność treningu:&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[wartość]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_27">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Poziom wydolności:&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[wartość]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="label_28">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Typ treningu:&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[wybór]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QComboBox" name="m_pTrainingType">
        <item>
         <property name="text">
          <string>Inny</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bieg w I zakresie</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bieg w II zakresie</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bieg w III zakresie</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Interwały</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Kros</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fartlek</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wybieganie</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wycieczka biegowa</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ćwiczenia siłowe</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Basen</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rytmy</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QSpinBox" name="m_pEnduranceLevel">
        <property name="minimum">
         <number>-30</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="m_pTrainingEffect"/>
      </item>
      <item row="1" column="7">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Odpoczynek:&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[h]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QSpinBox" name="m_pRestTime"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTableView" name="m_pIntervalView">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="m_pQuitButton">
     <property name="text">
      <string>Anuluj</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
